x/gov/types/v1beta1/codec.go

```diff
@@ -32,6 +33,10 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 		(*Content)(nil),
 		&TextProposal{},
 	)
+	registry.RegisterImplementations(
```

Comment:
are we fine with introducing this dependency?

Reply:
Gov already depends on distribution so it isn't a new dependency introduction.